discord-scripts/thread-management/check-thread-archiving.ts

```diff
@@ -63,7 +78,7 @@ function requestFollowUpAction(
       .followUp({
         content: `Thanks ${userMention(
           requestingUserId,
-        )}, please ${requestedAction} this thread or it will be archived in 24 hours ❤️`,
+        )}, please ${requestedAction} this thread or it will be archived in <t:${followUpDeadline}:F> (<t:${followUpDeadline}:R> ❤️)`,
```

Comment:
```suggestion
        )}, please ${requestedAction} this thread or it will be archived in <t:${followUpDeadline}:F> (<t:${followUpDeadline}:R>) ❤️`,
```